### DISCUSSION TITLE
Installation of WSLg added a new distro, now to activate it on old one?

### DISCUSSION THREAD

**Discussion by Yury-Fridlyand** [microsoft/wslg#859] [Q&A]
Hi!

I installed WSLg using Windows Update (it was proposed automatically). I discovered that UI application doesn't work on my distro, `/mnt/wslg/.X11-unix/` is empty, but ... I got a new distro with WSLg working.
Is it possible to install it on existing distro? I can't migrate and/or keep both.
`uname -a` on old one:
```
5.15.57.1-microsoft-standard-WSL2 #1 SMP Wed Jul 27 02:20:31 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```
on new:
```
5.15.57.1-microsoft-standard-WSL2 #1 SMP Wed Jul 27 02:20:31 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

Thanks in advance!

**Comment by crramirez** [comment]
Check this: https://github.com/microsoft/wslg/issues/844
And this: https://github.com/microsoft/wslg/issues/853

try to upgrade to:  WSL 0.70.0 to see if this fix you the error

**Comment by Yury-Fridlyand** [comment]
@crramirez,
Thank you for your reply. Unfortunately, this didn't help.
Is it possible to copy/migrate `WSLg` config or drivers from one distro to another?
